config/standard/resources.json

```diff
@@ -1,26 +1,25 @@
 {
    "area": "icon-places-CHW-area@2x.png",
-   "branch": "icon-places-clinic@2x.png",
+   "branch": "icon-places-clinic@2x.svg",
    "child": "icon-people-child@2x.png",
    "children": "icon-people-children@2x.png",
-   "clinic": "icon-places-clinic@2x.png",
-   "family": "icon-people-family@2x.png",
+   "clinic": "icon-places-clinic@2x.svg",
+   "family": "icon-people-family@2x.svg",
    "follow-up": "icon-followup-general@2x.png",
    "household": "icon-places-household@2x.png",
    "immunization": "icon-healthcare-immunization@2x.png",
    "infant": "icon-people-baby@2x.png",
    "mother-child": "icon-people-woman-baby@2x.png",
-   "new-person": "icon-people-person-general@2x.png",
+   "new-person": "icon-people-person-general@2x.svg",
    "nurse": "icon-people-nurse@2x.png",
    "off": "icon-messages-off@2x.png",
    "on": "icon-messages-on@2x.png",
-   "person": "icon-people-person-general@2x.png",
+   "person": "icon-people-person-general@2x.svg",
    "pregnancy-1": "icon-people-woman-pregnant@2x.png",
    "pregnancy-4": "icon-people-woman-pregnant@2x.png",
    "risk": "icon-healthcare-warning@2x.png",
    "chw": "icon-people-CHW-female@2x.png",
    "medic-clinic": "medic-clinic.svg",
    "medic-health-center": "medic-health-center.svg",
-   "medic-district-hospital": "medic-district-hospital.svg",
-   "medic-person": "medic-person.svg"
+   "medic-district-hospital": "medic-district-hospital.svg"
```

Comment:
I see you've deleted "medic-person" - are you sure it's never used?

Reply:
Oops!